tools/generator/cmd/track2/release/sdk.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I believe this is not track 2 particular definitions. Could we move this to a more common place?